src/app.js

```diff
@@ -1,12 +1,38 @@
 go.app = function() {
     var vumigo = require('vumigo_v02');
     var App = vumigo.App;
+    var _ = require('underscore');
+    var fs = require('fs');
     var Choice = vumigo.states.Choice;
     var ChoiceState = vumigo.states.ChoiceState;
+    var PaginatedChoiceState = vumigo.states.PaginatedChoiceState;
     var EndState = vumigo.states.EndState;
 
+
     var GoApp = App.extend(function(self) {
         App.call(self, 'states:start');
+        // var $ = self.$;
+
+        self.states.add('states:lookup', function(name) {
+            var provinces = self.im.sandbox_config.get("locations.json", {json: true});
+            provinces = JSON.parse(fs.readFileSync("src/lookups/provinces.json", "utf8"));
+            console.log("app.js");
+            console.log(provinces);
+            console.log("/app.js");
+            
+            province_array = [];
+            _.each(provinces, function(val) {
+                province_array.push(val.Province);
+            });
```

Comment:
Please also read http://vumi-jssandbox-toolkit.readthedocs.org/en/master/states/reference.html#PaginatedChoiceState more carefully. There are two options -- `options_per_page` and `characters_per_page`. You can either limit options per page or characters per page. You probably want characters per page. That requires `{options_per_page: null, characters_per_page: 160}`.